internal/tools/changes/cmd/changetool/release.go

```diff
@@ -20,10 +27,19 @@ func releaseUsage() {
 
 func init() {
 	updatePendingFlags = flag.NewFlagSet("update-pending", flag.ExitOnError)
+	updatePendingFlags.StringVar(&releaseParams.repo, "repo", "", "path to the SDK git repository")
 	updatePendingFlags.Usage = func() {
-		fmt.Printf("%s release update-pending <repo>\n  <repo>: path to git repository\n", os.Args[0])
+		fmt.Printf("%s release update-pending <repo>\n", os.Args[0])
 		updatePendingFlags.PrintDefaults()
 	}
+
+	staticVersionsFlags = flag.NewFlagSet("static-versions", flag.ExitOnError)
```

Comment:
Given the number of commands the tool will end up having, I'll probably create a more structured way to represent a subcommand than this. Maybe something like [how the go command does this](https://golang.org/pkg/cmd/go/internal/base/#Command).